code/game/machinery/kitchen/microwave.dm

```diff
@@ -253,38 +231,42 @@
 	if (!recipe)
 		dirty += 1
 		if (prob(max(10,dirty*5)))
-			if (!wzhzhzh(16))
-				abort()
+			if (!cook_for_time(16))
+				stop()
 				return
 			muck_start()
-			wzhzhzh(16)
+			cook_for_time(16)
 			muck_finish()
 			cooked = fail()
 			cooked.forceMove(src.loc)
 			return
 		else if (has_extra_item())
-			if (!wzhzhzh(16))
-				abort()
+			if (!cook_for_time(16))
+				stop()
 				return
 			broke()
 			cooked = fail()
 			cooked.forceMove(src.loc)
 			return
 		else
-			if (!wzhzhzh(40))
-				abort()
+			if (!cook_for_time(40))

```

Comment:
These are a lot of magic numbers. They should really be a define.

Reply:
sorted (probably) in next commit